HealthVault/Classes/Models/Generated/MHVTaskTrackingOccurrence.m

```diff
@@ -0,0 +1,82 @@
+//
+// MHVTaskTrackingOccurrence.m
+// MHVLib
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/**
+* NOTE: This class is auto generated by the swagger code generator program.
+* https://github.com/swagger-api/swagger-codegen.git
+* Do not edit the class manually.
+*/
+
+
+#import "MHVTaskTrackingOccurrence.h"
+
+@implementation MHVTaskTrackingOccurrence
+
++ (BOOL)shouldValidateProperties
+{
+    return YES;
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    // initialize property's default value, if any
+    
+  }
+  return self;
+}
+
+
+
+/**
+ * Maps json key to property name.
+ * This method is used by `JSONModel`.
+
++ (JSONKeyMapper *)keyMapper {
+  return [[JSONKeyMapper alloc] initWithModelToJSONDictionary:@{ @"taskId": @"taskId", @"trackingDateTime": @"trackingDateTime" }];
+}
+ */
```

Comment:
if JSONModel not used, could remove so it doesn't generate the commented out code